services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/RouteOptionsTest.java

```diff
@@ -387,7 +395,14 @@ private RouteOptions routeOptionsList() {
         add(0);
       }})
       .continueStraight(false)
-      .exclude(DirectionsCriteria.EXCLUDE_TOLL)
+      .excludeList(new ArrayList<String>() {{
+          add(DirectionsCriteria.EXCLUDE_TOLL);
+          add(DirectionsCriteria.EXCLUDE_TUNNEL);
```

Comment:
This test uses a `DirectionsCriteria.PROFILE_DRIVING_TRAFFIC` and excludes `TUNNEL`, but `TUNNEL` exclude is not mentioned in the documentation above. If `DRIVING` and `DRIVING_TRAFFIC` both support `TUNNEL` as exclude could you please add it to the documentation

Reply:
That is correct. That being said, Directions team is working on supporting the `DRIVING_TRAFFIC` profile in the short term https://github.com/mapbox/mapbox-navigation-android/issues/3844#issuecomment-743361069

In order to be more accurate, I can fix the tests to use `DRIVING` profile instead 👍

Reply:
Addressed this.

Reply:
Thanks for addressing the feedback, but I meant this:

```
A list of exclude. Exclude certain road types from routing.
     * The default is to not exclude anything from the profile selected.
     * The following exclude flags are available for each profile:
     * <p>
     * {@link DirectionsCriteria#PROFILE_DRIVING}: One of {@link DirectionsCriteria#EXCLUDE_TOLL},
     * {@link DirectionsCriteria#EXCLUDE_MOTORWAY}, or {@link DirectionsCriteria#EXCLUDE_FERRY}.
```
In the test you are using `EXCLUDE_TUNNEL` for `PROFILE_DRIVING`, but the documentation does not mention that it is supported. Could you add `{@link DirectionsCriteria#EXCLUDE_TUNNEL}` to the documenation?

Reply:
Going to fix the tests and use `ferry` for example. `tunnel` is not supported based on https://docs.mapbox.com/api/navigation/directions/

| Profile                | Available excludes                               |
|------------------------|--------------------------------------------------|
| mapbox/driving         | One or more of `toll`, `motorway`, `ferry`, or `unpaved` |
| mapbox/driving-traffic | One or more of `toll`, `motorway`, `ferry`, or `unpaved` |
| mapbox/walking         | No excludes supported                            |
| mapbox/cycling         | `ferry`                                            |

Thanks for confirming @danpat